package/upgrade/do_upgrade_node.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+until $SCRIPT_DIR/upgrade_node.sh $@; do
+  # Do not fail pre-drain/post-drain jobs
+  case $1 in
+    "pre-drain"|"post-drain")
+	;;
+    *)
+	exit 1
+	;;
+  esac
```

Comment:
We also call the `do_upgrade_node.sh` in the image preloading phase, but that argument (`prepare`) is not considered here. If the preload somehow failed, the wrapper will exit with return code 1. Is the design intended?

Reply:
I initially included the `prepare`. But considering it's still in the early stage and the user can start over and upgrade again, so it's excluded.

Reply:
That seems reasonable. Thanks.